truenas/webui
# NAS-111431 / 21.08 / Enclosures should listen for global resilver events (by dszidi)

Automatic cherry-pick failed. Please resolve conflicts by running:

    git reset --hard HEAD~1
    git cherry-pick -x a06f2ca4215e49e803c5727fe5460c57e1f4d61e

Testing:
- Remove a disk from a pool that has a spare
- Put the disk back
- Make sure the resilver icon shows up on the topbar
- The disk should reappear and should be presented as part of the pool again AFTER the resilver process completes

Original PR: https://github.com/truenas/webui/pull/5747
Jira URL: https://jira.ixsystems.com/browse/NAS-111431
